doc/libpmem2/pmem2_vm_reservation_map_find.3.md

```diff
@@ -21,54 +21,64 @@ date: pmem2 API version 1.0
 
 # NAME #
 
-**pmem2_vm_reservation_map_find**() - finds the mapping stored in the virtual
-memory reservation
+**pmem2_vm_reservation_map_find**(), **pmem2_vm_reservation_map_find_previous**(),
+**pmem2_vm_reservation_map_find_next**(), **pmem2_vm_reservation_map_find_first**() and
+**pmem2_vm_reservation_map_find_last**() - search for the mapping located at the
+desirable location
 
 # SYNOPSIS #
 
 ```c
 #include <libpmem2.h>
 
+struct pmem2_map;
 struct pmem2_vm_reservation;
 int pmem2_vm_reservation_map_find(struct pmem2_vm_reservation *rsv,
-		size_t reserv_offset, size_t len, struct pmem2_map **map);
-
-int pmem2_vm_reservation_map_find_closest_prior(
-		struct pmem2_vm_reservation *rsv, size_t reserv_offset,
-		size_t len, struct pmem2_map **map);
-
-int pmem2_vm_reservation_map_find_closest_later(
-		struct pmem2_vm_reservation *rsv, size_t reserv_offset,
-		size_t len, struct pmem2_map **map);
+		size_t reserv_offset, size_t len, struct pmem2_map **map_ptr);
+int pmem2_vm_reservation_map_find_previous(struct pmem2_vm_reservation *rsv,
+		struct pmem2_map *map, struct pmem2_map **map_ptr);
+int pmem2_vm_reservation_map_find_next(struct pmem2_vm_reservation *rsv,
+		struct pmem2_map *map, struct pmem2_map **map_ptr;
+int pmem2_vm_reservation_map_find_first(struct pmem2_vm_reservation *rsv,
+		struct pmem2_map **map_ptr);
+int pmem2_vm_reservation_map_find_last(struct pmem2_vm_reservation *rsv,
+		struct pmem2_map **map_ptr);
 ```
 
 # DESCRIPTION #
```

Comment:
this should probably define the order (what is first, what is next?), are the mappings in the insert order or the address order?